specs/latest/2.0/index.html

```diff
@@ -768,22 +827,51 @@ <h2 class="no-toc">Table of contents</h2>
+  void uniform4iv(WebGLUniformLocation? location, Int32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+
+  void uniform1fv(WebGLUniformLocation? location, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniform2fv(WebGLUniformLocation? location, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniform3fv(WebGLUniformLocation? location, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniform4fv(WebGLUniformLocation? location, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+
+  void uniform1uiv(WebGLUniformLocation? location, Uint32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniform2uiv(WebGLUniformLocation? location, Uint32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniform3uiv(WebGLUniformLocation? location, Uint32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniform4uiv(WebGLUniformLocation? location, Uint32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+
+  void uniformMatrix2fv(WebGLUniformLocation? location, GLboolean transpose, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniformMatrix3x2fv(WebGLUniformLocation? location, GLboolean transpose, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniformMatrix4x2fv(WebGLUniformLocation? location, GLboolean transpose, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+
+  void uniformMatrix2x3fv(WebGLUniformLocation? location, GLboolean transpose, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniformMatrix3fv(WebGLUniformLocation? location, GLboolean transpose, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniformMatrix4x3fv(WebGLUniformLocation? location, GLboolean transpose, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+
+  void uniformMatrix2x4fv(WebGLUniformLocation? location, GLboolean transpose, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniformMatrix3x4fv(WebGLUniformLocation? location, GLboolean transpose, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+  void uniformMatrix4fv(WebGLUniformLocation? location, GLboolean transpose, Float32Array data, optional unsigned long srcOffset = 0, optional unsigned long srcLength = 0);
+
```

Comment:
You can choose to upload less than the whole uniform location's size. For instance, for only updating one index in a uniform array.